src/Installers/Windows/HostOptions/Product.wxs

```diff
@@ -54,27 +54,15 @@
 
             <?define ProductOptionsKey=SOFTWARE\Microsoft\dotnet\host\options\$(var.MajorVersion).$(var.MinorVersion)?>
 
-            <Component Id="OPT" KeyPath="yes" Guid="{5FF00E0C-2BEE-42CA-8359-D46D6CB74DD1}">
+            <Component Id="OPT" Guid="*">
                 <RegistryKey Root="HKLM" Key="$(var.ProductOptionsKey)">
-                    <RegistryValue Name="OPT_NO_ANCM" Type="integer" Value="[OPT_NO_ANCM]" />
-                </RegistryKey>
-                <RegistryKey Root="HKLM" Key="$(var.ProductOptionsKey)">
-                    <RegistryValue Name="OPT_NO_FTS" Type="integer" Value="[OPT_NO_FTS]" />
-                </RegistryKey>
-                <RegistryKey Root="HKLM" Key="$(var.ProductOptionsKey)">
-                    <RegistryValue Name="OPT_NO_LTS" Type="integer" Value="[OPT_NO_LTS]" />
-                </RegistryKey>
-                <RegistryKey Root="HKLM" Key="$(var.ProductOptionsKey)">
-                    <RegistryValue Name="OPT_NO_SHAREDFX" Type="integer" Value="[OPT_NO_SHAREDFX]" />
-                </RegistryKey>
-                <RegistryKey Root="HKLM" Key="$(var.ProductOptionsKey)">
-                    <RegistryValue Name="OPT_NO_RUNTIME" Type="integer" Value="[OPT_NO_RUNTIME]" />
-                </RegistryKey>
-                <RegistryKey Root="HKLM" Key="$(var.ProductOptionsKey)">
-                    <RegistryValue Name="OPT_NO_X86" Type="integer" Value="[OPT_NO_X86]" />
-                </RegistryKey>
-                <RegistryKey Root="HKLM" Key="$(var.ProductOptionsKey)">
-                    <RegistryValue Name="OPT_NO_SHARED_CONFIG_CHECK" Type="integer" Value="[OPT_NO_SHARED_CONFIG_CHECK]" />
+                    <RegistryValue Name="OPT_NO_ANCM" Type="integer" Value="[OPT_NO_ANCM]" KeyPath="yes" />
```

Comment:
Only one item can be the keypath of a component. I'd be very surprised if this compiled. Your options are one of the registry values, or splitting each value into a separate component if you care about repairs